code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -24,6 +23,10 @@ var/const/SLIDE_ICE = 8
 
 
 /mob/living/carbon/Process_Spacemove(movement_dir = 0)
+	var/obj/item/device/flightpack/F = get_flightpack()
```

Comment:
Shouldn't this code be directly above the code for the other kinds of jetpacks, instead of above the parent call and loc check?